alpaca_trade_api/stream2.py

```diff
@@ -370,6 +378,19 @@ async def close(self, renew):
                                            self._oauth,
                                            raw_data=self._raw_data)
 
+    async def signal_stop_ws(self):
```

Comment:
[minor] I would simply call this `stop()` or `stop_ws()`

Reply:
done